server/webapp/WEB-INF/rails.new/app/assets/stylesheets/css/build_detail.scss

```diff
@@ -290,9 +290,37 @@ div.build_detail_container_content {
 .widget iframe {
     border: none;
 }
-#buildoutput_pre{
+#buildoutput_pre {
```

Comment:
Avoid using id selectors<br>Selector `buildoutput_pre` should be written in lowercase with hyphens